volmdlr/faces.py

```diff
@@ -8899,6 +8902,30 @@ def from_mesh_data(cls, positions, faces):
             triangles.append(Triangle3D(points[i1], points[i2], points[i3]))
         return cls(triangles)
 
+    def points_cloud_distances(self, points_cloud: List[volmdlr.Point3D]) \
```

Comment:
There are some point cloud classes in volmdlr/cloud.py
Why not using them?

Reply:
Done in last commit : Moved method to PointCloud3D and add a method to generate a trimesh from ClosedTriangleShell3D